mediawiki/issues.md

```diff
@@ -0,0 +1 @@
+[https://phabricator.wikimedia.org/maniphest/task/edit/form/1/?projects=MediaWiki-Docker](https://phabricator.wikimedia.org/maniphest/task/edit/form/1/?projects=MediaWiki-Docker)
```

Comment:
When I click this link, it prompts me to create an account or login -- shouldn't this link to a list of currently open issues instead so that folks can search before they file a new issue?